ansible/roles/debops.elasticsearch/defaults/main.yml

```diff
@@ -306,6 +306,21 @@ elasticsearch__jvm_memory_min_heap_size: '{{ (((ansible_memtotal_mb|int
 # This usually should be the same as the minimum heap size, for performance
 # reasons.
 elasticsearch__jvm_memory_max_heap_size: '{{ elasticsearch__jvm_memory_min_heap_size }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# Paths [[[
+# ------------------
```

Comment:
Can you write the section title underline flush with the title?

Reply:
Done.